index.html

```diff
@@ -5,8 +5,231 @@
+            <a class="navigation-link" href="#" >Доставка</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Гарантия</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Контакты</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <section class="gallery">
+          <h2 class="gallery-title">Фотогалерея</h2>
+          <img class="slider-image" src="" alt="Примеры" >
+          <button class="slider-button slider-prev" type="button">
+            <span class="visually-hidden">Предыдущее фото</span>
+          </button>
+          <button class="slider-button slider-next" type="button">
+            <span class="visually-hidden">Следующее фото</span>
+          </button>
+          <ol class="slider-pagination">
+            <li class="slider-pagination-item">
+              <button class="slider-pagination-button slider-pagination-button-current" type="button">
+                <span class="visually-hidden">1 фото</span>
+              </button>
+            </li>
+            <li class="slider-pagination-item">
+              <button class="slider-pagination-button" type="button">
+                <span class="visually-hidden">2 фото</span>
+              </button>
+            </li>
+            <li class="slider-pagination-item">
+              <button class="slider-pagination-button" type="button">
+                <span class="visually-hidden">3 фото</span>
+              </button>
+            </li>
+          </ol>
+        </section>
+        <section class="info-photo-gallery">
+          <h2 class="information-gallery">
+            Порхает как бабочка, жалит как пчела!
+          </h2>
+          <span class="information-small-gallery">
+            Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.
+          </span>
+          <ul class="information-button">
+            <li class="information-item">
+              <a class="nformation-button" href="#" >Подробнее</a>
+            </li>
+          </ul>
+          <h2 class="visually-hidden">Преимущества</h2>
+        <li class="Product-info">
+          <h3 class="Product Features">800 м</h3>
+          <p class="Product Features-info">Дальность полета</p>
+        </li>
+        <li class="Product-info">
+          <h3 class="Product Features">50 м</h3>
+          <p class="Product Features-info">Радиус поражения</p>
+        </li>
+        </section>
+      </div>
+    </header>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
-
+    <main class="main-index">
+      <section>
+    <h2 class="visually-hidden">Каталог</h2>
+  <li class="advantages">
+    <img src="" alt="VR-очки">
+    <p class="advantages-title">Виртуальная
+      реальность</p>
```

Comment:
```suggestion
    <p class="advantages-title">Виртуальная реальность</p>
```
В коде не нужно переносить по строкам, это будет делаться классами (ограничение ширины)